### ISSUE TITLE
Displays element and vertex numberings, but not dof numberings

### ISSUE THREAD

**Issue by rw-anderson** [GLVis/glvis#106]
It should probably display dof numberings as well.  Maybe something like this for Q2. :)

![dofs](https://user-images.githubusercontent.com/11507994/88258789-cbc2c280-cc75-11ea-99c0-45d7a0fd4a5f.png)

**Comment by tzanio** [comment]
I agree this could be useful :).

It could also be nice to be able to display the DOF numbering for the GridFunction that is being visualized.

**Comment by rw-anderson** [comment]
Yes, I'm grabbing the DOFs from the GridFunction.  Here's an L2 GridFunction.  Still some work to complete this.  I'll push a branch.

![star-L2](https://user-images.githubusercontent.com/11507994/88338709-9954ac00-cced-11ea-8219-80f6c3de5cce.png)

**Comment by tzanio** [comment]
ping: @rw-anderson

**Comment by tzanio** [comment]
@rw-anderson, do you want to add this as part of the upcoming `glvis-4.1` or should we postpone it to the next release?

**Comment by hughcars** [comment]
Just wanted to say this would be extremely helpful for debugging feature additions in mfem. Even more useful if it would be able to handle NC dofs.

**Comment by camierjs** [comment]
This issue should be resolved by the changes in PR #339, which adds support for displaying DOF numberings.
